sql/core/src/main/scala/org/apache/spark/sql/execution/adaptive/LocalShuffledRowRDD.scala

```diff
@@ -57,38 +57,20 @@ private final class LocalShuffledRowRDDPartition(
 class LocalShuffledRowRDD(
      var dependency: ShuffleDependency[Int, InternalRow, InternalRow],
      metrics: Map[String, SQLMetric],
-     advisoryParallelism : Option[Int] = None)
+     partitionStartIndices: Array[Array[Int]])
```

Comment:
Add these comments in the description ?

```
 * @param partitionStartIndices A mapper usually writes many shuffle blocks, and it's better to
 *                              launch multiple tasks to read shuffle blocks of one mapper. This
 *                              array contains the partition start indices for each mapper.
```

Reply:
We can follow https://github.com/apache/spark/blob/master/sql/core/src/main/scala/org/apache/spark/sql/execution/ShuffledRowRDD.scala#L107-L110